libs/community/langchain_community/llms/huggingface_pipeline.py

```diff
@@ -199,12 +199,20 @@ def _generate(
             batch_prompts = prompts[i : i + self.batch_size]
 
             # Process batch of prompts
-            responses = self.pipeline(
-                batch_prompts,
-                stop_sequence=stop,
-                return_full_text=False,
-                **pipeline_kwargs,
-            )
+
+            if self.pipeline.task == "text-generation":
```

Comment:
Isn't this change meant to be applied for "summarization" too?
```suggestion
            if self.pipeline.task in ("text-generation", "summarization"):
```